main.go

```diff
@@ -192,6 +192,7 @@ func init() {
 	execCmd.Flags().IntVar(&config.GPUMemory, "gpu-mem", 0, "Memory of GPU if applicable for a gpuType, e.g., 12")
 	execCmd.Flags().StringVar(&config.GPUType, "gpu-type", "", "Type of GPU to use, e.g., rtx_5000")
 	execCmd.Flags().IntVar(&config.CPUs, "cpus", 0, "Number of VCPUs to allocate, e.g., 4")
+	execCmd.Flags().StringVar(&config.CPUType, "cpu-type", "", "Type of CPU to use, e.g., xeon_v4")
```

Comment:
I'm not sure this is necessary tbh. It might be too much detail. Most other ML companies don't expose the type of CPU. We can do it internally but for the user we can just let them decide gpu (including type) or cpu (no type).